src/ngraph/op/topk.cpp

```diff
@@ -235,13 +236,16 @@ op::v1::TopK::TopK(const Output<Node>& data,
                    const element::Type& index_element_type)
     : Op{{data, k}}
     , m_axis{axis}
+    , m_normalized_axis{0}
```

Comment:
This needs to be a `op::v2::TopK` because the semantics have changed; even though it is backwards compatible for frontends, it is not backwards compatible for users of the op like passes and backends.

Reply:
I see your point but according to spec `TopK:v1` should handle negative axis. For me it was base to introduce this change.  Can we change something in order to provide compatibility?

Reply:
So it's bug-fix. Okay.